packages/core/src/components/SessionTimeout/SessionTimeout.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { Modal, Button, Col, Icon } from '../../index';
+
+const SessionTimeout = props => (
+  <Modal
+    show={props.secondsLeft > 0 && props.secondsLeft <= props.displayBefore}
+  >
+    <Modal.Body style={{ padding: '50px' }}>
+      <Col className="session-timeout-warning-icon" md={2}>
+        <Icon type="pf" name="warning-triangle-o" />
+      </Col>
+      <Col className="session-timeout-message" md={10}>
+        <p>Your session is about to expire.</p>
+        <p>
+          You will be logged out in{' '}
+          {props.secondsLeft > 0 ? props.secondsLeft : 0} seconds.
```

Comment:
is it always going to be a small amount of seconds? ( I wouldnt want to see you'll be logged out in 12317983 seconds )

Reply:
I added `props.units` and renamed `secondsLeft` to `timeLeft`. Hopefully, this is good enough.